main.go

```diff
@@ -159,8 +159,12 @@ func main() {
 		lexer = lexers.Get(config.Input)
 	}
 
-	if config.Language != "" {
-		lexer = lexers.Get(config.Language)
+	if config.Language[0] != "" && len(config.Language) == 1 {
+		lexer = lexers.Get(config.Language[0])
+	} else if len(config.Language) == 2 {
+		language := lexers.Get(config.Language[0])
+		root := lexers.Get(config.Language[1])
+		lexer = chroma.DelegatingLexer(root, language)
```

Comment:
I am using `chroma.DelegatingLexer` as the docs stated [here](https://github.com/alecthomas/chroma/blob/v2.13.0/delegate.go#L20). `Lexer` is an interface so all methods used for `lexer` should ideally work.

Reply:
My implementation is working. Need more test-cases/